proceds-blockly.js

```diff
@@ -1,774 +1,783 @@
-  Blockly.Msg.PROCEDURES_DEFNORETURN_PROCEDURE = "Hacer algo";
-  Blockly.Msg.PROCEDURES_DEFNORETURN_TITLE = "Definir";
-  Blockly.Msg.PROCEDURES_DEFNORETURN_NOPARAMS = "";
-  Blockly.Msg.PROCEDURES_DEFRETURN_NOPARAMS = "";
-  Blockly.Msg.PROCEDURES_DEFRETURN_COMMENT = 'Describe la función...';
-  Blockly.Msg.PROCEDURES_DEFRETURN_PROCEDURE = "devolver algo";
-  Blockly.Msg.PROCEDURES_DEFRETURN_TITLE = "Definir";
-  Blockly.Msg.PROCEDURES_BEFORE_PARAMS = "con";
-  Blockly.Msg.PROCEDURES_DEFNORETURN_TOOLTIP = "Crea un procedimiento.";
-  Blockly.Msg.PROCEDURES_DEFRETURN_TOOLTIP = "Crea una función.";
-  Blockly.Msg.PROCEDURES_ADD_PARAMETER = "Agregar parámetro";
-  Blockly.Msg.PROCEDURES_ADD_PARAMETER_PROMPT = "Ingresa el nombre del parámetro";
-  Blockly.Msg.PROCEDURES_REMOVE_PARAMETER = "Quitar parámetro";
-  Blockly.Msg.PROCEDURES_PARAMETER = "parámetro";
-
-  // -------------------------------------
-  // [!] Fixing FieldImage's click handler
-  // -------------------------------------
-
-  Blockly.FieldImage.prototype.init = function() {
-    if (this.fieldGroup_) {
-      // Image has already been initialized once.
-      return;
-    }
-    // Build the DOM.
-    /** @type {SVGElement} */
-    this.fieldGroup_ = Blockly.utils.createSvgElement('g', {}, null);
-    if (!this.visible_) {
-      this.fieldGroup_.style.display = 'none';
-    }
-    /** @type {SVGElement} */
-    this.imageElement_ = Blockly.utils.createSvgElement(
-      'image',
-      {
-        'height': this.height_ + 'px',
-        'width': this.width_ + 'px'
-      },
-      this.fieldGroup_);
-    this.setValue(this.src_);
-    this.sourceBlock_.getSvgRoot().appendChild(this.fieldGroup_);
-
-    // Configure the field to be transparent with respect to tooltips.
-    this.setTooltip(this.sourceBlock_);
-    Blockly.Tooltip.bindMouseEvents(this.imageElement_);
-
-    var self = this;
-
-    if (this.clickHandler_) { // [!]
-      this.imageElement_.addEventListener("mousedown", function(event) {
-        event.stopPropagation();
-        event.preventDefault();
-
-        if (event.button === 0) {
-          self.clickHandler_();
-        }
-      });
-      this.mouseDownWrapper_ =
-        Blockly.bindEventWithChecks_(
-            this.fieldGroup_, 'mousedown', this, this.clickHandler_);
-    }
-  };
-
-  // [!] Modifying callbackFactory to return the created block
-  Blockly.ContextMenu.callbackFactory = function(block, xml) {
-    return function() {
-      Blockly.Events.disable();
-      try {
-        var newBlock = Blockly.Xml.domToBlock(xml, block.workspace);
-        // Move the new block next to the old block.
-        var xy = block.getRelativeToSurfaceXY();
-        if (block.RTL) {
-          xy.x -= Blockly.SNAP_RADIUS;
-        } else {
-          xy.x += Blockly.SNAP_RADIUS;
-        }
-        xy.y += Blockly.SNAP_RADIUS * 2;
-        newBlock.moveBy(xy.x, xy.y);
-      } finally {
-        Blockly.Events.enable();
+const ProcedsBlockly = {
```

Comment:
Tengo la ligera sospecha de que la razón por la que lo otro estalla está acá.
Acá hay que poner:

```suggestion
window.ProcedsBlockly = {
```

Explicación acá:

https://eqsash.com/articles/chem-otlichaetsya-obyavlenie-peremennyh-v-javascript-chto-vybrat-window-var-let-const?l=en

Estamos usando este proyecto esperando justamente que agregue al espacio global el objeto ProcedsBlockly.